### ISSUE TITLE
Starting fluxbox and wpa-cute doesn't stay on tray.

### ISSUE THREAD

**Issue by kzapfe** [loh-tar/wpa-cute#9]
Hello loh-tar:
I am using Arch-GNU/Linux and have installed wpa-cute from AUR. I start fluxbox with a normal
"exec starfluxbox" on the .xinirc file and have the line 
`wpa-cute -t & `
in the startup file of the fluxbox.
But the wpa-cute doesn't stay. It opens its normal window but no sytray icon and if I close the window it is gone ( it works well anyway).
If i start it afterward with the "-t" option the icon appears, so that part seems to be working.

**Comment by kzapfe** [comment]
Yeah... maybe something with fluxbox, because that is the file that I am using.

**Comment by loh-tar** [comment]
Ah, misread your post, sorry.

Hm, you description give me the idea that wpaCute is started before fluxbox can handle these -t stuff. So, how about to start wpaCute with some delay, e.g by some helper script?

BTW the Wiki regarding autostart is for me slightly confusing, _The ~/.fluxbox/startup file is a script for autostarting applications as well as starting Fluxbox itself...Make sure that any lines starting applications come before the call to start Fluxbox itself._ So it is started twice? Once by .xinitrc and then again by autostart?

**Comment by Lightsockie** [comment]
Experiencing this as well with Awesome. Rather I add `wpa-cute -t &` to `.xinitrc` or to `.config/awesome/rc.lua` (what I assume is the equiv of Fluxbox's startup), I'm unable to get the icon to show up on the sys-tray. The program just opens normally, and if closed just closes entirely. It does work as expected if I manually run `wpa-cute -t` in a terminal tho. Using `dhcpcd-ui` via `.xinitrc` now (specifically `dhcpcd-gtk`), but its much more limited functionality-wise :'^). I've noticed that `wpa_supplicant_gui` seems to have the same issue, though I didn't investigate it quite as much.

Running Artix (btw)
```
$ pacman -Qi wpa-cute | grep Version
Version         : 0.8.6-1
$ awesome --version
awesome v4.3 (Too long)
```
(Unrelated, but minor feature request for a `--version` flag lol)

**Comment by loh-tar** [comment]
Hi!

the ony "new" idea I have is to try `(sleep 5; wpa-cute -t)&` in your `.xinitrc` As above said to delay the start by some helper script. 

ATM I'm not in the mood to investigate that by install fluxbox or awesome. IIRC, I looked at the wpaCute code at the time, but couldn't figure out where the problem might be. Next time I'm working on wpaCute I will look again... hopefully with more vigor.
